### PR TITLE
feat(detection): add container option

### PR DESCRIPTION
**Pull request by kminoda**
## Description

Add container option for 
- lidar_centerpoint
- image_projection_based_fusion/pointpainting
- euclidean_clustering

image

MUST BE MERGED WITH https://github.com/autowarefoundation/autoware_launch/pull/834

Note that the `inf` values are replaced with a large value, since it somehow fails with ComposableNode launch

## Tests performed

Confirmed that Autoware launches successfully with
- use_pointcloud_container = true
- use_pointcloud_container = false

and also verified that the result of `ros2 component list` is as expected.
For true: 
[use_pointcloud_container_true.txt](https://github.com/autowarefoundation/autoware.universe/files/14106133/use_pointcloud_container_true.txt)
 and for false: 
[use_pointcloud_container_false.txt](https://github.com/autowarefoundation/autoware.universe/files/14106135/use_pointcloud_container_false.txt)

## Effects on system behavior

When `use_pointcloud_container = false`, no effects are expected. When `use_pointcloud_container = true`, internal data communication will be reduced which may results in an efficient system performance.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: perception/lidar_centerpoint/launch/lidar_centerpoint.launch.xml [hunk 1]
```diff
@@ -11,20 +11,45 @@
...
+        <param name="build_only" value="$(var build_only)"/>
+        <param from="$(var model_param_path)"/>
+        <param from="$(var class_remapper_param_path)"/>
+      </composable_node>
+    </load_composable_node>
+  </group>
+  <group unless="$(var use_pointcloud_container)">
+    <node pkg="lidar_centerpoint" exec="lidar_centerpoint_node" name="lidar_centerpoint" output="screen">
+      <remap from="~/input/pointcloud" to="$(var input/pointcloud)"/>
+      <remap from="~/output/objects" to="objects"/>
```
[the remark this answers is not in the document]
Comment (kminoda) [reply perception/lidar_centerpoint/launch/lidar_centerpoint.launch.xml:40]: Thank you, fixed: https://github.com/autowarefoundation/autoware.universe/pull/6228/commits/704589bc29a3c04fd69e2022788567ff10c99376

[the remark this answers is not in the document]
Comment (kminoda) [reply perception/lidar_centerpoint/launch/lidar_centerpoint.launch.xml]: Right, misunderstood. Thanks. https://github.com/autowarefoundation/autoware.universe/pull/6228/commits/9238f2ccd23100cb9c702eadc1097994e37600d4

**Review by miursh — APPROVED** [review]
LGTM

**Comment by kminoda** [comment]
@yukkysaito @knzo25 Hi, would you review this PR? We need your approval for `euclidean_cluster` and `lidar_centerpoint`, respectively.

**Review by knzo25 — APPROVED** [review]
LGTM  :bow:

**Review by yukkysaito — APPROVED** [review]
LGTM

**Pull request merged**